targets/TARGET_Cypress/TARGET_PSOC6/psoc6csp/hal/src/cyhal_crc.c

```diff
@@ -36,19 +36,15 @@
 cy_rslt_t cyhal_crc_init(cyhal_crc_t *obj)
```

Comment:
The resource type changed from `CYHAL_RSC_CRC` to `CYHAL_CRYPTO_CRC`.

Is the CRC and Crypto accelerator the same hardware block? Should this be renamed to `cyhal_crypto_init()` now that it works with `CYHAL_CRYPTO_CRC`?

Reply:
CRC as other operations peprormed by the same hardware block

Reply:
OK